### PR TITLE
fix: fix gradle17 and add gradle11 for powertools

### PR DESCRIPTION
**Pull request by jeromevdl**
*Issue #, if available:*

*Description of changes:*

- Adding gradle for java 11 with powertools that is currently missing
- Fixing gradle for java 17 with powertools (aspectJ + gradle version)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: java11/hello-pt-gradle/{{cookiecutter.project_name}}/README.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+* [Visual Studio](https://docs.aws.amazon.com/toolkit-for-visual-studio/latest/user-guide/welcome.html)
+
+## Deploy the sample application
+
+The Serverless Application Model Command Line Interface (SAM CLI) is an extension of the AWS CLI that adds functionality for building and testing Lambda applications. It uses Docker to run your functions in an Amazon Linux environment that matches Lambda. It can also emulate your application's build environment and API.
+
+To use the SAM CLI, you need the following tools.
+
+* SAM CLI - [Install the SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
+* java17 - [Install the Java 17](https://docs.aws.amazon.com/corretto/latest/corretto-17-ug/downloads-list.html)
```
Comment (jfuss) [inline java11/hello-pt-gradle/{{cookiecutter.project_name}}/README.md:34]: Shouldn't this be Java11?

Comment (jeromevdl) [reply java11/hello-pt-gradle/{{cookiecutter.project_name}}/README.md:34]: good catch, updating

**Comment by jeromevdl** [comment]
I made the changes and a bit of cleanup in other versions too, plus upgraded to the latest version. Ready for review

**Review by jfuss — APPROVED** [review]

**Review by hnnasit — APPROVED** [review]
LGTM!

**Pull request merged**